src/assets/scss/ActivityCard.scss

```diff
@@ -106,3 +107,33 @@
 .activity__status--rejected::before {
   background-color: #b23030;
 }
+
+.activity-button {
+  display: block;
+  width: 6rem;
+  height: 3rem;
+  float: right;
+  position: relative;
+  opacity: 0.9;
+  padding: .3rem;
+  border: 0 Solid;
+  cursor: pointer;
+  border-radius: .4rem;
```

Comment:
I will prefer being more explicit here, we can just use `0.4` instead. This will boost consistency and better code understanding